proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -1069,14 +1068,22 @@ def _get_performance_contents(self, time_point=0):
         if hasattr(self, "velocity_io"):
             var_dict["Velocity @Inlet"] = self.velocity_io[time_point, 'in']
             var_dict["Velocity @Outlet"] = self.velocity_io[time_point, 'out']
-        var_dict['Concentration @Inlet,Membrane-Interface '] = (
-                    self.feed_side.properties_interface_in[time_point].conc_mass_phase_comp['Liq', 'NaCl'])
-        var_dict['Concentration @Outlet,Membrane-Interface '] = (
-            self.feed_side.properties_interface_out[time_point].conc_mass_phase_comp['Liq','NaCl'])
-        var_dict['Concentration @Inlet,Bulk'] = (
-                    self.feed_side.properties_in[time_point].conc_mass_phase_comp['Liq', 'NaCl'])
-        var_dict['Concentration @Outlet,Bulk'] = (
-            self.feed_side.properties_out[time_point].conc_mass_phase_comp['Liq','NaCl'])
+        for j in self.config.property_package.solute_set:
+            cin_mem_name=f'{j} Concentration @Inlet,Membrane-Interface '
+            var_dict[cin_mem_name] = (
+                        self.feed_side.properties_interface_in[time_point].conc_mass_phase_comp['Liq', j])
+            cout_mem_name=f'{j} Concentration @Outlet,Membrane-Interface '
+            var_dict[cout_mem_name] = (
+                self.feed_side.properties_interface_out[time_point].conc_mass_phase_comp['Liq', j])
+            cin_bulk_name=f'{j} Concentration @Inlet,Bulk '
+            var_dict[cin_bulk_name] = (
+                        self.feed_side.properties_in[time_point].conc_mass_phase_comp['Liq', j])
+            cout_bulk_name=f'{j} Concentration @Outlet,Bulk '
+            var_dict[cout_bulk_name] = (
+                self.feed_side.properties_out[time_point].conc_mass_phase_comp['Liq', j])
+            cp_name=f'{j} Permeate Concentration '
+            var_dict[cp_name] = (
+                self.permeate_side.properties_mixed[time_point].conc_mass_phase_comp['Liq', j])
```

Comment:
A nit -- I preferred the way you expressed this in the 1DRO model, without the intermediate variables.

Reply:
Oh, I still have to add that "has_full_reporting" functionality to 0DRO. It's on the list but would be in another PR.